ui/src/analyses/computers/computer_header/view.ts

```diff
@@ -0,0 +1,181 @@
+                    new SaveButton({onButtonClick: this.updateRun, parent: this.constructor.name}).render($)
+                    if (this.isUserLogged.is_user_logged && this.computer.is_claimed) {
+                        new DeleteButton({
+                            onButtonClick: this.onDelete.bind(this),
+                            parent: this.constructor.name
+                        }).render($)
+                    }
+                } else {
+                    new EditButton({onButtonClick: this.onToggleEdit, parent: this.constructor.name}).render($)
+                }
+            })
+            $('h2.header.text-center', 'Run Details')
+            $('div.input-list-container', $ => {
+                $('ul', $ => {
+                    this.nameField = new EditableField({
+                        name: 'Run Name',
+                        value: this.computer.name,
+                        isEditable: this.isEditMode
+                    })
+                    this.nameField.render($)
+                    this.commentField = new EditableField({
+                        name: 'Comment',
+                        value: this.computer.comment,
+                        isEditable: this.isEditMode
+                    })
+                    this.commentField.render($)
+                    $(`li`, $ => {
+                        $('span.item-key', 'Run Status')
+                        $('span.item-value', $ => {
+                            new StatusView({status: this.status.run_status}).render($)
+                        })
+                    })
+                    new EditableField({
+                        name: 'Computer UUID',
+                        value: this.computer.computer_uuid,
+                    }).render($)
+                    new EditableField({
+                        name: 'Session UUID',
+                        value: this.computer.session_uuid,
+                    }).render($)
+                    new EditableField({
+                        name: 'Start Time',
+                        value: formatTime(this.computer.start_time),
+                    }).render($)
+                    new EditableField({
+                        name: 'Last Recorded',
+                        value: this.status.isRunning ? getTimeDiff(this.status.last_updated_time * 1000) :
+                            formatTime(this.status.last_updated_time),
+                    }).render($)
+                })
+            })
+        })
+    }
+
+    onToggleEdit = () => {
+        this.isEditMode = !this.isEditMode
+
+        this.renderComputerHeader().then()
+    }
+
+    onDelete = async () => {
+        if (confirm("Are you sure?")) {
+            try {
+                await CACHE.getRunsList().deleteRuns(new Set<string>([this.uuid]))
+            } catch (e) {
+                handleNetworkError(e)
+                return
+            }
+            ROUTER.navigate('/runs')
```

Comment:
Shouldn't this go back to `'/computers'`  instead of runs ?

Reply:
changed thanks.